lib/plist/parser.rb

```diff
@@ -73,10 +71,10 @@ def initialize( plist_data_or_file, listener )
     end
 
     TEXT       = /([^<]+)/
-    XMLDECL_PATTERN = /<\?xml\s+(.*?)\?>*/um
-    DOCTYPE_PATTERN = /\s*<!DOCTYPE\s+(.*?)(\[|>)/um
-    COMMENT_START = /\A<!--/u
-    COMMENT_END = /.*?-->/um
+    XMLDECL_PATTERN = /<\?xml\s+(.*?)\?>*/m
```

Comment:
I removed the `u` because otherwise we wouldn't even be able to parse the `xml` declaration without this error.